### PR TITLE
add templates

### LINKED ISSUE #181: Add Issue and PR templates
I have contributed to FreeCodeCamp and they have a very nice [Issue](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/549cfc305f75d8c6036a3cde65c47b518607b18c/.github/ISSUE_TEMPLATE.md) and [PR](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/6ce5f14e779673ae117de41c664fea6989750bc9/.github/PULL_REQUEST_TEMPLATE.md) templates. 

And when PRs are merged, the corresponding Issue mentioned in the template is closed. I don't know how it happens, but somewhere it was mentioned that GitHub does it automatically. Do check out about it. 

And Issue template is very useful. You can ask if the person is a user/tester/ developer and you can ask the version of the App he/she is using, for testers(or) the branch he/she is on, for developers and ask them the issue's description, their OS details and then may be even a screenshot. You can add more to the template. The above was was just a suggestion. And yeah, this will make the issue more elaborate and you won't have to ask redundant questions to the people who report the issues.

### PR DESCRIPTION
**Pull request by ZakariaRidouh**
fixes #181 

review needed. maybe we can reword some words or add some new requirements

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#### Issue Description
+<!-- Describe below when the issue happens and how to reproduce it -->
+
+
+#### Browser Information
+<!-- Describe your workspace in which you are having issues-->
+
+* App Version:
+* Operating System:
+* Node Version
```
Comment (amilajack) [inline .github/ISSUE_TEMPLATE.md]: Would be great if you could add `npm version` to this as well

**Comment by amilajack** [comment]
What do you think about [eslint's](https://github.com/eslint/eslint) issue and PR templates?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## Issue Name
+<!-- Whats this issue about? -->
+
+#### Issue Description
+<!-- Describe below when the issue happens and how to reproduce it -->
+
+
+#### Browser Information
```
Comment (karuppiah7890) [inline .github/ISSUE_TEMPLATE.md]: Change the heading to `System Configuration` or `System Information`. Because we aren't talking about browsers here

**Comment by karuppiah7890** [comment]
@amilajack Yes. I think we can borrow the last part. Instead of asking them about the Issue description, and how to reproduce them, we could do ask these Qs : 

**What did you do?**
**What did you expect to happen?**
**What actually happened?**

**Comment by amilajack** [comment]
@karuppiah7890 Great idea. That would be so useful!

**Comment by karuppiah7890** [comment]
Yes ! :smile:

**Comment by amilajack** [comment]
@ZakariaRidouh Thanks! Would be great if you could integrate @karuppiah7890 's [recommendations](https://github.com/amilajack/popcorn-time-desktop/pull/182#issuecomment-252543203).

**Comment by ZakariaRidouh** [comment]
@amilajack @karuppiah7890 sure thing!

**Comment by ZakariaRidouh** [comment]
@amilajack @karuppiah7890 how does it look now?

**Comment by amilajack** [comment]
LGTM! Merging!

**Comment by karuppiah7890** [comment]
Ah, okay

**Comment by amilajack** [comment]
@ZakariaRidouh actually can you please make those changes! @karuppiah7890 sorry about that. Should have waited for your response

**Comment by amilajack** [comment]
@karuppiah7890 @ZakariaRidouh  also I would really appreciate it if you could add your own GitHub reviews to PR's (including my own)

**Comment by karuppiah7890** [comment]
It's fine. And I think we need to add another PR now for the small changes. And @ZakariaRidouh Do change the `Browser Information` heading to or `System and Environment Information`. @amilajack It sounds good ? Or you could suggest something better

And yes! I would love to review! :grin:

**Comment by amilajack** [comment]
@karuppiah7890 sounds good!

**Comment by karuppiah7890** [comment]
:+1: And by the way, I misunderstood "including my own" as including your own reviews. So, we are gonna review even your PRs ? That is some big stuff! We will try our best !

**Comment by amilajack** [comment]
Oh I meant 'including my own' as in mine! So I want you and @ZakariaRidouh to review PR's that I make.

**Comment by ZakariaRidouh** [comment]
@amilajack @karuppiah7890 fixed in #184

cheers!

**Comment by amilajack** [comment]
Thanks! @karuppiah7890 Would be great if you could look over it.

**Comment by karuppiah7890** [comment]
Sure, I am ready to review your PRs too !

**Pull request merged**